src/operations/expansions/expandGeom3.js

```diff
@@ -0,0 +1,23 @@
+const { geom3 } = require('../../geometry')
+
+const union = require('../booleans/union')
+
+const expandShell = require('./expandShell')
+
+/**
+ * Expand the given geometry (geom3) using the given options (if any).
+ * @param {Object} options - options for expand
+ * @param {Number} [options.delta=1] - delta (+/-) of expansion
+ * @param {Integer} [options.segments=defaultResolution3D] - number of segments when creating rounded corners
```

Comment:
the notion of resolution of expansion is very confusing (already was confusing in V1 )